MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -15,162 +15,87 @@ final class MovieQuizViewController: UIViewController, QuestionFactoryDelegate {
     }
     
-    private func showAnswerResult(isCorrect: Bool) {
-        yesClickedButton.isEnabled = false
+    func highlightImageBorder(isCorrectAnswer: Bool) {
+        imageView.layer.masksToBounds = true
+        imageView.layer.borderWidth = 8
+        imageView.layer.borderColor = isCorrectAnswer ? UIColor.ypGreen.cgColor : UIColor.ypRed.cgColor
         noClickedButton.isEnabled = false
-        
-        if isCorrect {
-            correctAnswers += 1
-            imageView.layer.masksToBounds = true
-            imageView.layer.borderWidth = 8
-            imageView.layer.borderColor = UIColor.ypGreen.cgColor
-            DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) { [weak self] in
-                guard let self = self else { return }
-                self.imageView.layer.borderWidth = 0
-                self.yesClickedButton.isEnabled = true
-                self.noClickedButton.isEnabled = true
-                self.showNextQuestionOrResults()
-            }
-        } else {
-            imageView.layer.borderWidth = 8
-            imageView.layer.borderColor = UIColor.ypRed.cgColor
-            
-            
-            DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) { [weak self] in
-                guard let self = self else { return }
-                self.imageView.layer.borderWidth = 0
-                self.yesClickedButton.isEnabled = true
-                self.noClickedButton.isEnabled = true
-                self.showNextQuestionOrResults()
-            }
-        }
+        yesClickedButton.isEnabled = false
     }
     
-    private func showNetworkError(message: String) {
-        activityIndicator.stopAnimating()
-        let viewModel = AlertModel(title: "Ошибка", message: message, buttonText: "Попробовать еще раз") { [weak self] in
-            guard let self = self else {return}
-            
-            self.currentQuestionIndex = 0
-            self.correctAnswers = 0
-            
-            self.questionFactory?.loadData()
-        }
-        alertPresenter?.show(alertModel: viewModel)
+    func showLoadingIndicator() {
+        activityIndicator.isHidden = false
+        activityIndicator.startAnimating()
     }
     
-    func didLoadDataFromServer() {
-        activityIndicator.startAnimating()
-        questionFactory?.requestNextQuestion()
+    func hideLoadingIndicator() {
+        activityIndicator.isHidden = true
+        activityIndicator.stopAnimating()
     }
     
-    func didFailToLoadData(with error: Error) {
-        showNetworkError(message: error.localizedDescription)
+    func showNetworkError(message: String) {
+        hideLoadingIndicator()
+        
+        let alertModel = AlertModel(
+                                    title: "Ошибка",
+                                    message: message,
+                                    buttonText: "Попробовать еще раз") { [weak self] in
+            guard let self = self else {return}
```

Comment:
Пробелы:
```suggestion
            guard let self = self else { return }
```
А еще можно опускать вторую часть, когда имена совпадают:
```suggestion
            guard let self else { return } 
```